### PR TITLE
Modify Reader:

### PR DESCRIPTION
**Pull request by pipu**
1. Parsing "FlipX", "FlipY", "AnchorPoint".
2. Parsing "VisibleForFrame" instead of "Visible"

### CODE REVIEW

File: cocos/editor-support/cocostudio/ActionTimeline/CSLoader.cpp [hunk 1]
```diff
@@ -1595,11 +1595,11 @@ void CSLoader::setPropsForNodeFromXML(cocos2d::Node *node, const tinyxml2::XMLEl
         }
         else if (name == "Visible")
         {
-            node->setVisible((value == "True") ? true : false);
+//            node->setVisible((value == "True") ? true : false);
```
Comment (minggo) [inline cocos/editor-support/cocostudio/ActionTimeline/CSLoader.cpp]: If `Visible` is not supported, please remove all the codes.
And is there compatibility issue?

**Comment by pipu** [comment]
I will modify it immediately.

File: cocos/editor-support/cocostudio/ActionTimeline/CSLoader.cpp [hunk 2]
```diff
@@ -1587,7 +1587,7 @@ void CSLoader::setPropsForNodeFromXML(cocos2d::Node *node, const tinyxml2::XMLEl
         }
         else if (name == "Rotation")
         {
-//            node->setRotation(atoi(value.c_str()));
+
         }
```
Comment (minggo) [inline cocos/editor-support/cocostudio/ActionTimeline/CSLoader.cpp]: The same issue.
And please modify all of them.

I will not mention it one by one.

**Comment by pipu** [comment]
Similar code as this line:
"//            node->setRotation(atoi(value.c_str()));"
I've deleted.
at above commit "Deleting original code that parsing "Visible""

**Comment by minggo** [comment]
[ci rebuild]

**Pull request merged**